.circleci/config.yml

```diff
@@ -163,7 +163,7 @@ commands:
 jobs:
   checks:
     docker:
-      - image: circleci/rust:latest
+      - image: cimg/rust:1.60.0
```

Comment:
Using `cimg` here upgrades us to using CircleCI's new build images (`circleci` uses the legacy build images)